nginx.conf

```diff
@@ -0,0 +1,26 @@
+server {
+  listen 80;
+  root /var/www;
+
+  location = /inbox/ {
+    alias /var/www/inbox/;
```

Comment:
not an nginx expert here. Are you sure you don't have to tell nginx that the name of the default file to lookup in a folder is `index.html` ?

Reply:
no since we will proxy pass to this one, we wont handle regex locations/ index here

Reply:
I mean the nginx equivalent (if needed) of https://httpd.apache.org/docs/2.4/mod/mod_dir.html#directoryindex